pdfbox/src/main/java/org/apache/pdfbox/pdmodel/font/encoding/MacOSRomanEncoding.java

```diff
@@ -27,7 +27,9 @@ public class MacOSRomanEncoding extends MacRomanEncoding
     /**
      * Table of octal character codes and their corresponding names
      * on top of {@link MacRomanEncoding}.
+     * ToDo are these really octals?? leading 0 seems to be missing
```

Comment:
Good catch!

Reply:
Thanks, I've added the leading "0", see PDFBOX-5773